language/e2e-tests/src/tests/genesis.rs

```diff
@@ -58,3 +61,27 @@ fn execute_genesis_write_set() {
     // Executing the genesis transaction should success.
     assert!(!executor.execute_transaction(txn).status().is_discarded());
 }
+
+#[test]
+fn initial_move_version() {
```

Comment:
Nit: this looks like more than a test for checking the initial version; it's also checking that updating the version works. Might might sense to rename + move outside of genesis.rs (if there a test file for on-chain configs)?